### PR TITLE
fix(ecam): Hide REFUELG immediately once refueling is complete

### LINKED ISSUE #5072: REFUELG stays until EFB page is opened
### Aircraft Version

Development

### Build info

```json
{
  "built": "2021-06-08T22:19:29+00:00",
  "ref": "merge",
  "sha": "de36fa54256cd283d95ab521dcdef07303fa5e88",
  "actor": "aguther",
  "event_name": "manual"
}
```

### Describe the bug

When fueling the plane via the EFB fueling function, REFUELG will be displayed on the ECAM memo. But even when finished fueling, it will stay until you click on the EFB fuel page once and it will update and disappear.

### Expected behavior

It should disappear shortly after fueling is finished.

### Steps to reproduce

1. Refuel a bit via EFB
2. Watch ECAM memo
3. Click EFB fuel page and it will update

### References (optional)

_No response_

### Additional info (optional)

_No response_

### Discord Username (optional)

_No response_

### PR DESCRIPTION
**Pull request by BlueberryKing**
Fixes #5072 

## Summary of Changes
Previosly, the REFUELG message would stay on the ECAM if one started the refueling process on the EFB but then changed to a different page during the refueling process. The only way to make the message disappear was to go to the Fuel page on the EFB. This has been fixed now.

## Additional context

Discord username (if different from GitHub): BlueberryKing#6641

## Testing instructions
- Spawn in cold & dark. There should be about 9.5t of fuel on board.
- Start the refueling process from the EFB, but switch to a different page such as Dashboard.
- Monitor fuel level on the ECAM. Ensure the REFUELG message disappears once the desired fuel level has been reached.
- Try this with different refueling speeds as well as defueling

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

File: flybywire-aircraft-a320-neo/html_ui/Pages/A32NX_Core/A32NX_Refuel.js [hunk 1]
```diff
@@ -163,5 +163,8 @@ class A32NX_Refuel {
                 return;
             }
         }
+
+        // DONE FUELING
+        SimVar.SetSimVarValue("L:A32NX_REFUEL_STARTED_BY_USR", "Bool", false);
```
Comment (tracernz) [inline flybywire-aircraft-a320-neo/html_ui/Pages/A32NX_Core/A32NX_Refuel.js:168]: Not a good idea to write this every update if it's already false (writing local vars is a little costly).

Comment (BlueberryKing) [reply flybywire-aircraft-a320-neo/html_ui/Pages/A32NX_Core/A32NX_Refuel.js:168]: We check whether this is set to false and return early at the start of the update method, so this should be fine, no?

Comment (tracernz) [reply flybywire-aircraft-a320-neo/html_ui/Pages/A32NX_Core/A32NX_Refuel.js:168]: Ah, my bad, you are correct. 👍

**Review by tracernz — APPROVED** [review]

**Comment by AviationSpotter** [comment]
Trainee Report

Discord: Aviation Spotter#9102
Object of testing: #5075
Tier of Testing: 1
Date: 10/6/2021

Testing Process: 

1. Spawned C&D
2. Refueled while on the Dashboard page and monitoring upper ECAM
3. Defueled while on the Dashboard page and monitoring upper ECAM
4. Repeated steps 2 and 3 with the refuel time options at Fast and Instant

Negatives: N/A
Testing Results: Passed

**Comment by beheh** [comment]
@BlueberryKing Thanks for your first contribution! Could you please update add a line to CHANGELOG.md, so we can merge this?

**Comment by BlueberryKing** [comment]
Alright, changelog should be updated now. :)

**Pull request merged**